tests/run_selective.py

```
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Ruff (0.8.2)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)